src/main/xerus/monstercat/api/Player.kt

```diff
@@ -167,31 +167,23 @@ object Player: FadingHBox(true, targetHeight = 25) {
 		}
 	}
 	
-	private val pauseButton = ToggleButton().id("play-pause")
-			.onClick { if (isSelected) player?.pause() else player?.play() }
-			.apply { tooltip = Tooltip("Pause / Play") }
+	private val pauseButton = ToggleButton().id("play-pause").onClick {
+		if (isSelected) player?.pause() else player?.play()
+	}.apply { tooltip = Tooltip("Pause / Play") }
 	private val stopButton = buttonWithId("stop") {
 		reset()
 		Playlist.clear()
-	}.apply {
-		tooltip = Tooltip("Stop playing")
-	}
+	}.apply { tooltip = Tooltip("Stop playing") }
 	private val volumeSlider = Slider(0.0, 1.0, Settings.PLAYERVOLUME()).scrollable(0.05).apply {
 		prefWidth = 100.0
 		valueProperty().listen { updateVolume() }
-	}.apply {
-		tooltip = Tooltip("Volume")
-	}
+	}.apply { tooltip = Tooltip("Volume") }
```

Comment:
double apply? Btw maybe we should have a `Control.tooltip` extension function, similar to the `.id`

Reply:
there is already one ^^

Reply:
We don't use it so often so I don't think it poses a real interest.

Reply:
Fixed double apply in 7660b501f1ce1fa0c1038089fc246e23af288433